### PR TITLE
display the bash commands that we're running

### PR DESCRIPTION
**Pull request by komidore64**

### CODE REVIEW

**Comment by dustints** [comment]
code looks good. :+1: 
going to run a quick test.

**Comment by dustints** [comment]
ACK. tested looks good...log output: https://gist.github.com/1223f595a516ff50f397

File: setup.rb [hunk 1]
```diff
@@ -12,6 +12,17 @@
   "2.1" => "releases/1.7"
 }
 
+# outputs system command as well as running the command
+module Kernel
+  def system_with_debug(*args)
+    puts "$ #{args.join(" ")}"
```
Comment (ehelms) [inline setup.rb]: What do you think about something more explicit (and less confusing with something manually from from the command line) such as "Running step: "

Comment (komidore64) [inline setup.rb]: `Running:` is cool. i'm down

Comment (komidore64) [inline setup.rb]: addressed

**Comment by komidore64** [comment]
@ehelms ?

**Comment by ehelms** [comment]
Looks good - only thing I might suggest is outputing a blank line before each command to make it easier to visually find where a new command is run. ACK either way.

**Comment by komidore64** [comment]
easy enough. i'm not against that.

**Comment by komidore64** [comment]
@ehelms updated

**Comment by ehelms** [comment]
ACK

**Pull request merged**